### ISSUE TITLE
Protobuf v1.5.2 is having End of Life

### ISSUE THREAD

**Issue by Kiran-38** [chartmuseum/helm-push#217]
Hi @jdolitsky @cbuto, as part of the security scan done we see there is a indirect package used in the helm-push is having EOF 

github.com/golang/protobuf,v1.5.2 --> There is a latest version 1.5.4 available.

**Comment by scbizu** [comment]
Yes , I see . But it seems that helm-push should support helm v2 which contains the EOL protobuf for compatibility . Maybe we can cut a legacy rc and release a new version without Helm v2 . 

/cc @cbuto @jdolitsky 

Anyway , thanks for remind us of this @Kiran-38

**Comment by Kiran-38** [comment]
Thank you @scbizu, can you please update and let me know when can we have a latest release with the fix. Any tentative date would be much appreciated.
